Mathlib/Algebra/Group/Units/Defs.lean

```diff
@@ -489,6 +492,14 @@ theorem unit_spec (h : IsUnit a) : ↑h.unit = a :=
 theorem unit_one (h : IsUnit (1 : M)) : h.unit = 1 :=
   Units.eq_iff.1 rfl
 
+@[to_additive (attr := simp)]
+theorem unit_mul (ha : IsUnit a) (hb : IsUnit b) : ha.unit * hb.unit = (ha.mul hb).unit :=
```

Comment:
Either this should be called `unit_mul_unit`, or it should be stated in reverse (and then use `simp<-` if you're really sure you want to simplify in this direction.

Reply:
(same for all of these)